modules/admin_manual/pages/configuration/server/excluded_blacklisted_files.adoc

```diff
@@ -123,7 +126,12 @@ But not:
 /etc/stuff/mydir1
 ----
 
-Example `blacklisted_files` entries in `config.php` look like this:
+=== Reasons for Blacklisting Files
+
+The reason for blacklisting files can be explained easily based on the example of a `.htaccess` file.
```

Comment:
Please don't use words such as "easy" or "easily", as what may be easy for you may not be so for another person.